amundsen_application/static/js/components/DashboardPage/ShimmeringDashboardLoader/styles.scss

```diff
@@ -1,26 +1,33 @@
-@import 'variables';
+@import "variables";
 
 $loading-duration: 1s;
-$gradient-start-color: $gray5;
-$gradient-end-color: $gray15;
+$loading-curve: cubic-bezier(0.45, 0, 0.15, 1);
 $shimmer-border-radius: 4px;
```

Comment:
Seems like the shimmer has a border-radius and the loaded preview image does not. I'm fine either way, but I think they should be consistent.

Reply:
Good call, the design's didn't have a border radius, I will remove it.

Thanks!